src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -320,7 +331,11 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
     return funcHasRecursiveCall(thread, implementation, funcIndex);
   }
 
-  render() {
+  renderContextMenuContents() {
+    if (!this.state.isShown) {
+      return null;
+    }
```

Comment:
for the sake of nitpicking, I think I'd put this check in `render` instead. I think this makes it easier to read, as `renderContextMenuContents` would always render the content, but we take the decision to call it in `render`.

Reply:
Yeah, that's clearer.